pkg/occlient/occlient.go

```diff
@@ -678,26 +679,38 @@ func (c *Client) GetLatestBuildName(buildConfigName string) (string, error) {
 }
 
 // StartBinaryBuild starts new build and streams dir as source for build
-func (c *Client) StartBinaryBuild(name string, dir string) error {
+// asFile indicates if it is a binary build or not
+func (c *Client) StartBinaryBuild(name string, dir string, asFile bool) error {
```

Comment:
`dir` parameter can no longer be called `dir`, as it is not always a directory, but it can be a path to a single file.
 Let's call it `path`